core/src/main/java/org/apache/accumulo/core/master/thrift/FateOperation.java

```diff
@@ -34,13 +34,14 @@
   TABLE_MERGE(6),
   TABLE_DELETE_RANGE(7),
   TABLE_BULK_IMPORT(8),
-  TABLE_COMPACT(9),
-  TABLE_IMPORT(10),
-  TABLE_EXPORT(11),
-  TABLE_CANCEL_COMPACT(12),
-  NAMESPACE_CREATE(13),
-  NAMESPACE_DELETE(14),
-  NAMESPACE_RENAME(15);
```

Comment:
Need to put the new type at the END of the list so the existing numbers don't change.